api/tests/test_hyp3_api.py

```diff
@@ -184,16 +184,13 @@ def test_bad_granule_names(client):
     response = submit_job(client, 'S1A_IW_GRDH_1SSH_20150609T141945_20150609T142014_006297_008439_B83')
```

Comment:
this was testing a name that was good except for being one character too short, should update to a truncated IW_SLC name

Reply:
`S1B_IW_SLC__1SDV_20200604T082207_20200604T082234_021881_029874_5E3`